app/routes/web.php

```diff
@@ -1,98 +1,33 @@
-Route::get('location-suggestions-for-name/{location_name}', 'LocationManagementController@getLocationSuggestionsForLocationName');
-Route::get('delete-my-location/{location_id}', 'LocationManagementController@deleteMyLocation');
-Route::get('user-report/{user_id}', 'InternalFeaturesController@showUserReport');
-Route::post('api/user/delete', 'InternalFeaturesController@deleteUser');
-Route::get('users', 'InternalFeaturesController@showUsers');
-Route::get('location-groups', 'InternalFeaturesController@showLocationGroupsReport');
-Route::get('dashboard', 'InternalFeaturesController@showDashboard');
-Route::get('map-location-visualizer', 'InternalFeaturesController@showMapVisualizer');
 Route::get('duplicate-location-finder', 'DuplicateLocationFinderController@showDuplicateLocationFinder');
-Route::get('location-report/{location_id}', 'LocationReportController@show');
-Route::get('location-report/{location_id}/{rating_system}', 'LocationReportController@show');
-Route::get('location-comments/{location_id}', 'LocationReportController@showComments');
-Route::get('location-map/{location_id}', 'LocationReportController@showMap');
-Route::get('location-map/{location_id}/{rating_system}', 'LocationReportController@showMap');
-Route::get('location-comprehensive-ratings/{location_id}', 'LocationReportController@showComprehensiveRatings');
-Route::get('location-comprehensive-ratings/{location_id}/{rating_system}', 'LocationReportController@showComprehensiveRatings');
-Route::get('location-reporting/{location_id}/{question_category_id}', 'LocationReportController@show2');
-Route::get('location-rating/{location_id}', 'LocationRatingController@show');
-Route::get('location-rating/{location_id}/{question_category_id}', 'LocationRatingController@show');
-Route::get('reviewed-locations', 'LocationRatingController@reviewedLocations');
-Route::post('location-rating-submit', 'LocationRatingController@submit');
-Route::put('location-rating/answer', 'LocationRatingController@setAnswer');
-Route::delete('location-rating/answer', 'LocationRatingController@removeAnswer');
-Route::put('location-rating/comment', 'LocationRatingController@setComment');
-Route::post('location-rating-commit', 'LocationRatingController@commitReview');
-Route::get('question-explanation/{question_id}', 'QuestionExplanationController@show');
-Route::get('api/question-explanation/{question_id}', 'QuestionExplanationController@getExplanation');
-Route::get('api/is-using-screen-reader', 'QuestionExplanationController@isUsingScreenReader');
 Route::post('time-zone', 'TimeZoneController@setTimeZone');
 
-Route::get('contact', 'ContactController@index');
-Route::post('contact', 'ContactController@sendMessage');
-Route::get('signin', 'SignInController@showForm');
-Route::post('signin', 'SignInController@authenticate');
-Route::get('signout', 'SignInController@signout');
+/* Authentication Routes */
+require "auth.php";
```

Comment:
I like what you put in each file.  The routes are grouped nicely.

We don't use 'require' much in other parts of the application, though.  Even though that's easy to understand, I'd rather include the other route files the same way that web.php is included.

Move the require statements to app/Providers/RouteServiceProvider.php 's mapWebRoutes() and mapApiRoutes():
![image](https://user-images.githubusercontent.com/11838191/40579736-8183c606-60fc-11e8-9a54-a0fa4c948783.png)

Referencing the route files from there would be more consistent with how Laravel's RouteServiceProvider initially tries to handle grouping the routes into different files.  It looks like I overlooked this earlier or I would have moved the 'api/' prefixed routes to routes/api.php.  I forget if I ran into any technical challenge with that.

There's some mention of this suggestion at: https://hisman.co/article/split-laravel-routes-into-multiple-files/